Misc/NEWS.d/next/Core_and_Builtins/2024-08-12-10-02-40.gh-issue-122928.e6_yVr.rst

```diff
@@ -0,0 +1 @@
+Make thread state(tstate) pointer safe
```

Comment:
No need for a NEWS entry. First, it's only guaranteed in debug builds, second, it was a pre-condition.